src/constants/index.ts

```diff
@@ -70,59 +70,69 @@ export const BLOCK_TIME = 13000 // ms
 
 export const BRIDGE_CONTRACT_ADDRESSES: { [chainId in ChainId]: string } = {
   [ChainId.MAINNET]: "0xf5059a5D33d5853360D16C683c16e67980206f36", // TODO replace once mainnet deploy goes out
+  [ChainId.ROPSTEN]: "0xf5059a5D33d5853360D16C683c16e67980206f36", // TODO couldn't find this
```

Comment:
@penandlim j/w has this been deployed to ropsten?

Reply:
No it hasn't yet. Due to the nature of virtual swaps and how we are using fake tokens in ropsten, we might have to test this in prod.